src/test/java/yevhenii/lostfilmdemo/convertors/TVSeriesConvertorTest.java

```diff
@@ -0,0 +1,37 @@
+package yevhenii.lostfilmdemo.convertors;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import yevhenii.lostfilmdemo.entity.FeedMessage;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest
+class TVSeriesConvertorTest {
```

Comment:
instead of setting active.profile=test in main property file add @ActiveProfile annotation for this test